.github/workflows/phpstan.yml

```diff
@@ -44,51 +57,19 @@ jobs:
     name: Static Analyse
     runs-on: ${{ inputs.runner }}
     steps:
-      - uses: shopware/github-actions/shopware-version@main
-        with:
-          fallback: ${{ inputs.shopwareVersionFallback }}
-          repo: ${{ inputs.shopware-repository }}
-        id: version
-        if: ${{ inputs.shopwareVersion == '.auto' || inputs.shopwareVersion == '' }}
       - name: Setup Shopware
-        uses: shopware/setup-shopware@main
+        uses: shopware/github-actions/setup-extension@main
         with:
-          shopware-version: ${{ steps.version.outputs.shopware-version || inputs.shopwareVersion }}
+          extensionName: ${{ inputs.extensionName }}
+          shopwareVersion: ${{ inputs.shopwareVersion }}
+          shopwareVersionFallback: ${{ inputs.shopwareVersionFallback }}
           shopware-repository: ${{ inputs.shopware-repository }}
-          composer-root-version: ${{ inputs.composerRootVersion }}
-          php-version: 8.2
+          composerRootVersion: ${{ inputs.composerRootVersion }}
+          phpVersion: 8.2
           env: prod
           install: "true"
-      - name: Clone Extension
-        uses: actions/checkout@v4
-        with:
-          path: custom/plugins/${{ inputs.extensionName }}
-      - name: Clone Dependencies
-        if: inputs.dependencies
-        env:
-          DEPS: ${{ inputs.dependencies }}
-        run: |
-          IFS=$'\n'
-          for token in $(echo "${{secrets.env}}"); do
-            export $token
```

Comment:
Hi @pweyck don't we need to export the variables in setup-extension.yaml? 
Because when cloning SwagCommercial, we need secrets.COMMERCIAL_CLONE_TOKEN

See cloning failure here:
 https://github.com/shopware/SwagCustomizedProducts/actions/runs/14260193709/job/39969945726?pr=85

```
for token in $(echo "${{secrets.env}}"); do
    export $token
done
```